dev-python/python-markdown-math/python-markdown-math-0.8-r1.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{9..11} pypy3 )
+inherit distutils-r1
+
+DESCRIPTION="Math extension for Python-Markdown"
+HOMEPAGE="https://github.com/mitya57/python-markdown-math"
+SRC_URI="mirror://pypi/${PN::1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~riscv ~x86"
+
+RDEPEND="
+	>=dev-python/markdown-3.3.7[${PYTHON_USEDEP}]
+"
+
+distutils_enable_tests setup.py
```

Comment:
Can you run tests via `unittest`? Using `setup.py test` is deprecated for some time already.

Reply:
The tests are successfully run with `distutils_enable_tests unittest` both for python 3.10 and 3.11 with the folowing output (slightly differ):
```
>>> Test phase: dev-python/python-markdown-math-0.8-r1
 * python3_10: running distutils-r1_run_phase python_test
python3.10 -m unittest_or_fail discover -v
test_begin_end (test.MathTestCase) ... ok
test_begin_end_preview (test.MathTestCase) ... ok
test_inline_asciimath (test.MathTestCase) ... ok
test_inline_gitlab (test.MathTestCase) ... ok
test_inline_inside_code (test.MathTestCase) ... ok
test_inline_inside_standalone (test.MathTestCase) ... ok
test_inline_latex (test.MathTestCase) ... ok
test_inline_latex_escaped (test.MathTestCase) ... ok
test_inline_latex_preview (test.MathTestCase) ... ok
test_inline_tex (test.MathTestCase) ... ok
test_inline_tex_disabled (test.MathTestCase) ... ok
test_inline_tex_escaped (test.MathTestCase) ... ok
test_standalone_gitlab (test.MathTestCase) ... ok
test_standalone_gitlab_nested (test.MathTestCase) ... ok
test_standalone_latex (test.MathTestCase) ... ok
test_standalone_latex_escaped (test.MathTestCase) ... ok
test_standalone_latex_preview (test.MathTestCase) ... ok
test_standalone_tex (test.MathTestCase) ... ok

----------------------------------------------------------------------
Ran 18 tests in 0.111s

OK
 * python3_11: running distutils-r1_run_phase python_test
python3.11 -m unittest_or_fail discover -v
test_begin_end (test.MathTestCase.test_begin_end) ... ok
test_begin_end_preview (test.MathTestCase.test_begin_end_preview) ... ok
test_inline_asciimath (test.MathTestCase.test_inline_asciimath) ... ok
test_inline_gitlab (test.MathTestCase.test_inline_gitlab) ... ok
test_inline_inside_code (test.MathTestCase.test_inline_inside_code) ... ok
test_inline_inside_standalone (test.MathTestCase.test_inline_inside_standalone) ... ok
test_inline_latex (test.MathTestCase.test_inline_latex) ... ok
test_inline_latex_escaped (test.MathTestCase.test_inline_latex_escaped) ... ok
test_inline_latex_preview (test.MathTestCase.test_inline_latex_preview) ... ok
test_inline_tex (test.MathTestCase.test_inline_tex) ... ok
test_inline_tex_disabled (test.MathTestCase.test_inline_tex_disabled) ... ok
test_inline_tex_escaped (test.MathTestCase.test_inline_tex_escaped) ... ok
test_standalone_gitlab (test.MathTestCase.test_standalone_gitlab) ... ok
test_standalone_gitlab_nested (test.MathTestCase.test_standalone_gitlab_nested) ... ok
test_standalone_latex (test.MathTestCase.test_standalone_latex) ... ok
test_standalone_latex_escaped (test.MathTestCase.test_standalone_latex_escaped) ... ok
test_standalone_latex_preview (test.MathTestCase.test_standalone_latex_preview) ... ok
test_standalone_tex (test.MathTestCase.test_standalone_tex) ... ok

----------------------------------------------------------------------
Ran 18 tests in 0.113s

OK
>>> Completed testing dev-python/python-markdown-math-0.8-r1
```